generatebundlefile/data/input_121.yaml

```diff
@@ -23,3 +23,10 @@ packages:
         registry: public.ecr.aws/l0g8r8j6
         versions:
             - name: latest
+  - org: emissary 
+    projects:
+      - name: emissary 
+        repository: emissary-ingress/emissary
+        registry: public.ecr.aws/l0g8r8j6
+        versions:
+            - name: latest
```

Comment:
Make sure to configure your IDE to always add newline at EOF.

Reply:
Will look into that